src/linter/root.go

```diff
@@ -698,7 +698,12 @@ func (d *RootWalker) handleFuncStmts(params []meta.FuncParam, uses, stmts []node
 		prematureExitFlags = cleanFlags
 	}
 
+	// if the function contains only one statement and this statement is throw.
+	oneThrowStatementFunction := b.containsThrow && len(stmts) == 1
```

Comment:
Why not two? Ten? Why this special case only?

Reply:
in `basic_test.go`:

```
// Functions of this kind are very common in Symfony, so it makes sense not to consider this 
// a loss of return, although in fact the @return annotation is not entirely correct.
```